characters/turkeyleg.py

```diff
@@ -0,0 +1,49 @@
+import os
+
+import pygame
+
+from utils.sprite import Sprite, Collider
+from utils.collision import CollisionEffect
+
+
+IMAGE_PATH = os.path.join('images', 'turkeyleg.png')
+IMAGE = pygame.image.load(IMAGE_PATH).convert_alpha()
+
+
+class TurkeyLeg(Sprite):
+    def __init__(self, x, y, angle, rotational_velocity):
+        super().__init__(
+            image=IMAGE,
```

Comment:
what is this, GNU style???

Reply:
:stuck_out_tongue:

Reply:
my pep8 linter complained until i did it!